buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -133,6 +133,7 @@ object TestDependencies {
     const val JaxbApi = "javax.xml.bind:jaxb-api:$JaxbApiVersion"
     const val XmlUnitCore = "org.xmlunit:xmlunit-core:$XmlUnitVersion"
     const val XmlUnitMatchers = "org.xmlunit:xmlunit-matchers:$XmlUnitVersion"
+    const val SnakeYaml = "org.yaml:snakeyaml"
```

Comment:
Should we set the specific version?

Reply:
no, it resolved by spring boot so in that way we use the same version as them and there is not risk of imcompability. Actually I was having some issues when using the last version.